src/backend/effects/builtin/activity-feed-alert.ts

```diff
@@ -0,0 +1,62 @@
+import { EffectType } from "../../../types/effects";
+import { EffectCategory } from "../../../shared/effect-constants";
+import { handleTriggeredEvent } from "../../events/activity-feed-manager";
+
+const effect: EffectType<{
+    message: string;
+    icon: string;
+}> = {
+    definition: {
+        id: "firebot:activity-feed-alert",
+        name: "Activity Feed Alert",
+        description: "Display an alert in Firebot's activity feed",
+        icon: "fad fa-comment-exclamation",
+        categories: [EffectCategory.FUN],
+        dependencies: []
+    },
+    optionsTemplate: `
+    <eos-container>
+        <p>Use this effect to send yourself alerts in Firebot's activity feed. This alert is only visible to you if the activity feed is visible.</p>
+    </eos-container>
+    <eos-container header="Alert Icon" pad-top="true">
+        <h3>Icon</h3>
+        <p class="muted">A custom icon which allows you to identify your Activity alert.</p>
+        <input maxlength="2" type="text" class="form-control" ng-model="effect.icon" icon-picker required>
+    </eos-container>
+    <eos-container header="Alert Message" pad-top="true">
+        <textarea ng-model="effect.message" class="form-control" name="text" placeholder="Enter message" rows="4" cols="40" replace-variables></textarea>
+    </eos-container> 
```

Comment:
Couple notes here:
- Since the icon is optional and the least important field here, I think it should be moved after the Message section to de-emphasize it. 
- I don't think there is a need for two different Icon headers, so we can remove the `<h3>` in favor of the built in one from `<eos-container>`.
- Instead of the `<textarea>`, we can use a `<firebot-input>` component

Here's what all those tweaks would look like if you want an easy copy/paste:
```html
<eos-container header="Message" pad-top="true">
        <firebot-input 
            model="effect.message" 
            placeholder-text="Enter message"
            use-text-area="true"
            rows="4"
            cols="40"
        />
</eos-container> 
<eos-container header="Icon" pad-top="true">
        <input 
			maxlength="2" 
			type="text" 
			class="form-control" 
			ng-model="effect.icon" 
			icon-picker required
		/>
</eos-container>
```